soql-environment/src/main/scala/com/socrata/soql/environment/AbstractName.scala

```diff
@@ -2,8 +2,8 @@ package com.socrata.soql.environment
 
 import com.ibm.icu.text.{Normalizer2, Normalizer}
 
-abstract class AbstractName[Self <: AbstractName[Self]](val name: String) extends Ordered[Self] {
-  final lazy val caseFolded = AbstractName.caseFold(name)
+abstract class AbstractName[Self <: AbstractName[Self]](val qualifier: Option[String], val name: String) extends Ordered[Self] {
+  final lazy val caseFolded = AbstractName.caseFold(toString)
```

Comment:
I don't think ColumnName should grow a qualifier on its own; I think the qualifier should live in ColumnRef instead and keep ColumnName as just a case-insensitive identifier.